src/main/scala/beam/utils/mapsapi/bingapi/BingExampleOfUsage.scala

```diff
@@ -0,0 +1,35 @@
+package beam.utils.mapsapi.bingapi
```

Comment:
This seems a bit hidden - should it be referenced in the docs somewhere?

Reply:
we already have two other examples: here-api, bing-api and google-api. we probable should define and create another task for updating all of them